horovod/runner/mpi_run.py

```diff
@@ -110,13 +119,15 @@ def _get_mpi_implementation(env=None):
 
 def _get_mpi_implementation_flags(tcp_flag, env=None):
     if is_open_mpi(env):
```

Comment:
@tgaddair why are all these `list()` around those `*_FLAGS`? They are lists already.

Reply:
Good question, I am not sure.  We can clean that up in a follow-up PR.